data/json/npcs/talk_tags.json

```diff
@@ -841,7 +1068,18 @@
       "Fight or flight?",
       "Weapons hot!",
       "Lock and load!",
-      "We've got incoming!"
+      "We've got incoming!",
+      "Yo, eyes open!",
+      "Alright, what the hell is going on?",
+      "Shit, <name_g>, we might have trouble here!",
+      "Things just went from bad to worse!",
+      "We're screwed man, screwed!",
+      "<swear!>, today was going so well too.",
+      "Looks like a storm's a'brewin!",
+      "We got contacts!",
+      "You see that?  What the hell?",
+      "Looks like trouble.",
+      "Aw <swear>, this doesn't look good."
```

Comment:
`this does not look good` maybe?

Reply:
I think this line is fine, however adding another line like;

"Aw <swear>, this does NOT look good!", how do you feel about that? Worth adding alongside this one?